sharepoint/sharepoint-ps/sharepoint-online/Export-SPODataAccessGovernanceInsight.md

```diff
@@ -0,0 +1,73 @@
+---
+external help file: Microsoft.Online.SharePoint.PowerShell.dll-Help.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version:
+schema: 2.0.0
+author: pvrk
+ms.author: pvrk
```

Comment:
>pvrk [](http://example.com/codeflow?start=11&length=4)

this should be your Microsoft alias. This file and others.

Reply:
When I updated with my Microsoft alias, it said that it is not the right Github alias.